sections/about-section.liquid

```diff
@@ -0,0 +1,200 @@
+{{ 'custom.css' | asset_url | stylesheet_tag }}
+<script src="{{ 'custom.js' | asset_url }}" defer></script>
```

Comment:
Since `custom.css` and `custom.js` are files that have rules that are shared across all of these sections, there's no need to add it to each section. You can just add it once to `theme.liquid` and it will be shared across the whole site. 

Adding it to each section leads to the same 2 files being loaded multiple times. You can see it if you check inspector. You'll see the css & js file being loaded nested under each section of the homepage. 

Tip: CSS files should go in the <head> section of theme.liquid. JS files should go before the closing of the </body> tag. 

*Caveat*: Use case when you would want to add the css and/or js file to each section is if you have CSS and/or JS that is specific to that particular section. I usually like to just include it in the .liquid file but some companies/devs might prefer the CSS & JS be separated out